ci/docker/debian-10-cpp.dockerfile

```diff
@@ -73,7 +73,9 @@ RUN apt-get update -y -q && \
 
 COPY ci/scripts/install_minio.sh \
      /arrow/ci/scripts/
-RUN /arrow/ci/scripts/install_minio.sh ${arch} linux latest /usr/local
+RUN if [ "${arch}" != "i386" ]; then \
+      /arrow/ci/scripts/install_minio.sh ${arch} linux latest /usr/local; \
```

Comment:
Can't find a binary for i386 here: https://dl.min.io/server/minio/release/

Possibly need to set `ARROW_S3=OFF` as well.

Reply:
Probably because nobody cares :-)

Reply:
Instead of putting the `if` here, put it in the download script?